src/ext/handlersToResp.c

```diff
@@ -25,6 +25,7 @@ typedef struct RedisToRdbVersion {
 } RedisToRdbVersion;
 
 const RedisToRdbVersion redisToRdbVersion[] = {
+        {"7.2", VER_VAL(7,2), 12}, // TODO: Truly support 12 and update released redis version.
```

Comment:
i fear it is all too easy to forget about these. both the opcodes we already have in unstable, and any that we'll have later in 8.0 (we won't have the rdb version bump as a trigger to handle them).

i suppose we can split such opcodes into two categories:
1. new opcodes for existing types, i.e. all of the sudden hash keys are encoded differently, and all tests will fail.
2. new opcodes that are used only in a specific case (like the cluster ones added in unstable).

maybe we can think of some way to have a test that greps all the opcodes in the redis source code and validates they're all supported in librdb? i.e. just match the opcodes / numbers, doesn't test their functionality.

anyway, that's just some thought for the future. not a blocker for this PR

Reply:
I already pushed a [commit](https://github.com/redis/librdb/pull/22/commits/ee0f60d2fd2fa2f89b9f332ed009c17407626403) that support the new opcode. I also modified the line to be  {"99.99", VER_VAL(99,99), 12}, to highlight that this value is not yet set. It doesn't have any side effect, just for the completeness.

Reply:
Lookup for function `elementSlotInfo()`. `99.99` tells (at least to me) that it is WIP.
I will consider how we can memcmp and alert on changes in opcodes by reading unstable/rdb.h.